packages/jira-adapter/src/filters/issue_layout/issue_layout.ts

```
@@ -113,20 +115,64 @@ const getProjectToScreenMapping = async (elements: Element[]): Promise<Record<st
   return projectToScreenId
 }
 
+const deployIssueLayoutChanges = async (
+  change: Change<InstanceElement>,
+  client: JiraClient,
+): Promise<void> => {
+  const issueLayout = getChangeData(change)
+  const items = issueLayout.value.issueLayoutConfig.items.map((item: IssueLayoutConfigItem) => {
+    if (isResolvedReferenceExpression(item.key)) {
+      const key = item.key.value.value.id
+      return {
+        type: item.type,
+        sectionType: item.sectionType.toLocaleLowerCase(),
+        key,
+        data: {
+          name: item.key.value.value.name,
+          type: item.key.value.value.type ?? item.key.value.value.schema.system,
+        },
+      }
+    }
+    return undefined
+  }).filter(isDefined)
+
+  if (isResolvedReferenceExpression(issueLayout.value.projectId)
+  && isResolvedReferenceExpression(issueLayout.value.extraDefinerId)) {
+    const data = {
+      projectId: issueLayout.value.projectId.value.value.id,
+      extraDefinerId: issueLayout.value.extraDefinerId.value.value.id,
+      issueLayoutType: 'ISSUE_VIEW',
+      owners: [],
+      issueLayoutConfig: {
+        items,
+      },
+    }
+    if (isAdditionChange(change)) {
+      const response = await getIssueLayout({ projectId: issueLayout.value.projectId.value.value.id,
+        screenId: issueLayout.value.extraDefinerId.value.value.id,
+        client })
+      if (!isIssueLayoutResponse(response.data)) {
+        return undefined
```

You should return an error to the user in this case